auth/auth_user.py

```diff
@@ -9,20 +9,19 @@
 
 
 class AuthUser:
-	phantom_set = None
 	user_profile = None # Cache the user
-
+	phantom_ids = []
```

Comment:
lets adjust this to be a dictionary, unless order here is required, if order does not matter then lets us a dictionary